web/src/components/contactForm.js

```diff
@@ -87,48 +87,53 @@ export default function ContactForm({ content }) {
                         </label>
                     </StyledHoneyPot>
                     <InputWrapper>
-                        <div>
-                            <StyledLabel htmlFor="firstName">
-                                First Name
-                                <StyledInput
-                                    type="text"
-                                    name="name"
-                                    id="firstname"
-                                    onChange={handleChange}
-                                    required
-                                />
-                            </StyledLabel>
-                            <StyledLabel htmlFor="lastName">
-                                Last Name
-                                <StyledInput
-                                    type="text"
-                                    name="name"
-                                    id="lastname"
-                                    onChange={handleChange}
-                                />
-                            </StyledLabel>
-                        </div>
-                        <div>
-                            <StyledLabel htmlFor="email">
-                                Email
-                                <StyledInput
-                                    type="email"
-                                    name="email"
-                                    id="email"
-                                    onChange={handleChange}
-                                    required
-                                />
-                            </StyledLabel>
-                            <StyledLabel htmlFor="title">
-                                Title
-                                <StyledInput
-                                    type="text"
-                                    name="title"
-                                    id="title"
-                                    onChange={handleChange}
-                                />
-                            </StyledLabel>
-                        </div>
+                        <StyledLabel htmlFor="firstName">
+                            First Name
+                            <StyledInput
+                                type="text"
+                                name="name"
+                                id="firstname"
+                                onChange={handleChange}
+                                required
+                            />
+                        </StyledLabel>
+                        <StyledLabel htmlFor="lastName">
+                            Last Name
+                            <StyledInput
+                                type="text"
+                                name="name"
+                                id="lastname"
+                                onChange={handleChange}
+                            />
+                        </StyledLabel>
```

Comment:
I didn't notice this before but I think you need to fix the casing for you `htmlFor` attribute (or change the `id` to match the `htmlFor`). Also you've used the same value (`name="name"`) for both name inputs. Need to change to something unique so netlify recognizes the different fields.